Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -4891,17 +4891,17 @@ PDFPluginHUDEnabled:
 
 PageAtRuleSupportEnabled:
   type: bool
-  status: unstable
+  status: stable
   category: css
   humanReadableName: "@page CSS at-rule support"
   humanReadableDescription: "Enable @page support"
   defaultValue:
     WebKitLegacy:
-      default: false
+      default: true
     WebKit:
-      default: false
+      default: true
     WebCore:
-      default: false
+      default: true
```

Comment:
Can we just get rid of this flag if it does nothing? or at least rename it to be accurate.

I recall Webkit has bugs where the `@page` descriptors are exposed as global CSS properties... which confirms to me that this pref doesn't do what it's supposed to do.